### PR TITLE
chore(maintenance): add Banxware customer refernece

### LINKED ISSUE #4840: [Support Lambda Powertools]: Banxware
### Organization Name

Banxware

### Your Name

Claudio Busatto

### Your current position

Lead Engineer

### (Optional) Company logo

https://uploads-ssl.webflow.com/630cd549c8022462080977f6/66a11836906d71b80e57e62f_Banxware%20Logo.svg

### (Optional) Use case

We’ve been using the [Event Handler features for REST API](https://docs.powertools.aws.dev/lambda/python/latest/core/event_handler/api_gateway/#api-gateway-rest-api) extensively in our Python services. Since we’re using the lambdalith approach, adding Powertools to our Python tech stack was a great move. It resolved multiple discussions within the engineering team, such as how to validate input data, how to log, how to trace, etc., in a simpler way. Sometimes, it's just a decorator, and we have the feature activated! Plus, it’s helped us establish a standard for writing our service code, thanks to the guidelines from the developer kit.

### Also using other Powertools for AWS Lambda languages?

- [ ] Java
- [X] TypeScript
- [ ] .NET

### PR DESCRIPTION
**Pull request by dreamorosi**
**Issue number:** closes #4840

### Changes

> Please provide a summary of what's being changed

The PR adds Banxware as customer reference according to the linked issue.

While working on this, I also noticed and fixed two things:
- The customer Alma Media was present as reference in the docs but not in the readme file
- Some of the `rel` attributes in the links of the customer references were incomplete

### User experience

> Please share what the user experience looks like before and after this change

N/A

## Checklist

If your change doesn't seem to apply, please leave them unchecked.

* [x] [Meet tenets criteria](https://docs.powertools.aws.dev/lambda/python/#tenets)
* [x] I have performed a self-review of this change
* [ ] Changes have been tested
* [x] Changes are documented
* [x] PR title follows [conventional commit semantics](https://github.com/aws-powertools/powertools-lambda-python/blob/develop/.github/semantic.yml)

<details>
<summary>Is this a breaking change?</summary>

**RFC issue number**:

Checklist:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

</details>

## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "develop"
+    ]
+}
```
Comment (leandrodamascena) [inline .vscode/settings.json]: .gitignore should prevent this file from being included. I don't know what happened here.

Please remove this file.

**Review by leandrodamascena — CHANGES_REQUESTED** [review]
Thanks for sending this PR @dreamorosi! Only one small fix.

File: README.md [hunk 2]
```diff
@@ -69,6 +69,8 @@ The following companies, among others, use Powertools:
 * [Transformity](https://transformity.tech/)
 * [Trek10](https://www.trek10.com/)
 * [Vertex Pharmaceuticals](https://www.vrtx.com/)
+* [Alma Media](https://www.almamedia.fi/en/)
+* [Banxware](https://www.banxware.com/)
```
Comment (leandrodamascena) [inline README.md]: We try to keep the alphabetical order. Can you please change this?

File: docs/index.md [hunk 3]
```diff
@@ -472,7 +472,10 @@ Knowing which companies are using this library is important to help prioritize t
 [**Vertex Pharmaceuticals**](https://www.vrtx.com/){target="_blank" rel="nofollow"}
 { .card }
 
-[**Alma Media**](https://www.almamedia.fi/en/){target="_blank" rel="nofollow}
+[**Alma Media**](https://www.almamedia.fi/en/){target="_blank" rel="nofollow"}
+{ .card }
+
+[**Banxware**](https://www.banxware.com){target="_blank" rel="nofollow"}
```
Comment (leandrodamascena) [inline docs/index.md]: We try to keep the alphabetical order. Can you please change this?

**Review by leandrodamascena — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 3 above)
Comment (dreamorosi) [reply docs/index.md]: Alma Media was already last - do you want me to put it at the top?

File: .vscode/settings.json (hunk 1 above)
Comment (dreamorosi) [reply .vscode/settings.json]: image

it was ignored - not sure either

Comment (dreamorosi) [reply .vscode/settings.json]: either way, I removed it.

**Comment by dreamorosi** [comment]
Sorted names in alphabetical order & removed the extra file.

**Review by leandrodamascena — APPROVED** [review]

**Pull request merged**